client/post-editor/media-modal/detail/detail-fields.jsx

```diff
@@ -80,6 +80,10 @@ class EditorMediaModalDetailFields extends Component {
 		return getMimePrefix( this.props.item ) === prefix;
 	}
 
+	isVideoPress() {
+		return this.getItemValue( 'videopress_guid' ) ? true : false;
```

Comment:
We could cast the value to boolean instead of using a ternary condition.

```suggestion
		return Boolean(this.getItemValue( 'videopress_guid' ));
```

In react is common to see a double negation too:
```js
return !! this.getItemValue( 'videopress_guid' );
```

Reply:
Thanks, double negation looks cool, I've used this one.